docs/src/api/class-keyboard.md

```diff
@@ -108,34 +108,44 @@ An example to trigger select-all with the keyboard
 await page.keyboard.press('Control+A');
 // on macOS
 await page.keyboard.press('Meta+A');
+// cross-platform
+await page.keyboard.press('ControlOrMeta+A');
```

Comment:
lets move it up two positions or nuke Control/Meta entirely.

Reply:
💣'd it